content/docs/deployment/csmoperator/_index.md

```diff
@@ -24,16 +24,16 @@ The table below lists the driver and modules versions installable with the CSM O
 
 | CSI Driver         | Version | CSM Authorization | CSM Replication | CSM Observability | CSM Resiliency |
 | ------------------ |---------|-------------------|-----------------|-------------------|----------------|
-| CSI PowerScale     | 2.7.0   | ✔ 1.7.0           | ✔ 1.5.0        | ✔ 1.5.0           | ✔ 1.6.0       |
+| CSI PowerScale     | 2.7.0   | ✔ 1.8.0           | ✔ 1.5.0        | ✔ 1.5.0           | ✔ 1.7.0       |
```

Comment:
For csm operator, v2.7.0 of the drivers will still support auth 1.7.0 and resiliency 1.6.0. A new line should be added for v2.8.0 of the drivers which will support the newer images. Operator supports n-2 versions, so v2.5.0 rows can be removed where applicable.